workflow/rocoto_funcs/base.py

```diff
@@ -76,6 +78,7 @@ def header_entities(xmlFile,expdir):
   <envar><name>HOMErrfs</name><value>&HOMErrfs;</value></envar>\n\
   <envar><name>EXPDIR</name><value>&EXPDIR;</value></envar>\n\
   <envar><name>DATAROOT</name><value><cyclestr>&DATAROOT;/{net}/{rrfs_ver}/{run}.@Y@m@d/@H</cyclestr></value></envar>\n\
+  <envar><name>UNBRELLA_DATA</name><value><cyclestr>&DATAROOT;/{net}/{rrfs_ver}/{run}.@Y@m@d/@H/{wgf}</cyclestr></value></envar>\n\
```

Comment:
`UNBRELLA_DATA` to `UMBRELLA_DATA` and other locations

Reply:
Done